README.md

```diff
@@ -22,109 +22,89 @@ Azure Thunderbolt is designed to run batch Spark jobs that require additional on
 ```bash
     pip install -e .
 ```
-4. Rename 'secrets.cfg.template' to 'secrets.cfg' and fill in the fields for your Batch account and Storage account.
+4. Initialize the project in a directory [This will automatically create a *.thunderbolt* folder with config files in your working directory]:
+```bash
+    azb spark init
+```
+5. Fill in the fields for your Batch account and Storage account in your *.thunderbolt/secrets.yaml* file. (We'd also recommend that you enter SSH key info in this file)
 
    Thunerbolt is built on top of two core Azure services, [Azure Batch](https://azure.microsoft.com/en-us/services/batch/) and [Azure Storage](https://azure.microsoft.com/en-us/services/storage/). Create those resources via the portal (see [Getting Started](./docs/00-getting-started.md)).
 
 ## Quickstart Guide
 
-The entire experience of this package is centered around a few commands.
+The core experience of this package is centered around a few commands.
+
+```sh
+# create your cluster
+azb spark cluster create
+```
+```sh
+# monitor and manage your clusters
+azb spark cluster get
+azb spark cluster list
+azb spark cluster delete
+```
+```sh
+# interact with your cluster
+azb spark cluster ssh
+azb spark cluster submit
```

Comment:
I think having this under 'interact with you cluster' is a bit misleading. This (for now) is the meat and potatoes, so maybe the title should be "log in or submit jobs to your cluster"